asserts/database.go

```diff
@@ -148,10 +148,23 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 
 	trustedBackstore := NewMemoryBackstore()
 
-	for _, accKey := range cfg.TrustedKeys {
-		err := trustedBackstore.Put(AccountKeyType, accKey)
-		if err != nil {
-			return nil, fmt.Errorf("error loading for use trusted account key %q for %q: %v", accKey.PublicKeyID(), accKey.AuthorityID(), err)
+	for _, a := range cfg.Trusted {
+		switch a.Type() {
```

Comment:
`switch v := a.(type)` would mean there's no need for the "cast" inside the case blocks, right?

However, I wonder if it would be simpler to just error if a.Type().Name is not in a list of trusted types? That would avoid any per-type code I think.